.github/workflows/main.yml

```diff
@@ -2,169 +2,167 @@ name: CI
+        with:
+          command: fmt
+          args: --all -- --check
+      - name: Lint with clippy
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: --workspace --all-targets --all-features -- -D warnings
 
   test:
     # Build & Test runs on all platforms
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         include:
-        - os: ubuntu-latest
-        - os: macOS-latest
+          - os: ubuntu-latest
+          - os: macOS-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Cache Rust dependencies
-      uses: actions/cache@v1.1.2
-      with:
-        # There's a problem with caching serde, hence we exclude it here
-        path: |
-          target
-          !target/**/*serde*
-        key: ${{ runner.OS }}-build-v2-${{ hashFiles('**/Cargo.lock') }}
-    - name: Install Mac System dependencies
-      if: startsWith(matrix.os,'macOS')
-      run: |
-        brew install boost
-    - name: Install Linux dependencies
-      if: startsWith(matrix.os,'ubuntu')
-      run: |
-        "${GITHUB_WORKSPACE}/.github/install_deps.sh"
-    - name: Install rust
-      uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        toolchain: stable
-        override: true
-    - name: Build
-      run: cargo build --all-features --verbose
-    - name: Run tests
-      run: cargo test --workspace --features solc-backend --verbose
-
-  wasm-test:
-      runs-on: ubuntu-latest
-      container: davesque/rust-wasm
-      steps:
       - uses: actions/checkout@v2
+      - name: Cache Rust dependencies
+        uses: actions/cache@v1.1.2
+        with:
+          # There's a problem with caching serde, hence we exclude it here
+          path: |
+            target
+            !target/**/*serde*
+          key: ${{ runner.OS }}-build-v2-${{ hashFiles('**/Cargo.lock') }}
+      - name: Install Mac System dependencies
+        if: startsWith(matrix.os,'macOS')
+        run: |
+          brew install boost
+      - name: Install Linux dependencies
+        if: startsWith(matrix.os,'ubuntu')
+        run: |
+          "${GITHUB_WORKSPACE}/.github/install_deps.sh"
       - name: Install rust
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: stable
           override: true
-      - name: Run WASM tests
-        run: wasm-pack test --node -- --workspace
+      - name: Build
+        run: cargo build --all-features --verbose
+      - name: Run tests
+        run: cargo test --workspace --features solc-backend --verbose
 
+  # wasm-test:
+  #     runs-on: ubuntu-latest
+  #     container: davesque/rust-wasm
+  #     steps:
+  #     - uses: actions/checkout@v2
+  #     - name: Install rust
+  #       uses: actions-rs/toolchain@v1
+  #       with:
+  #         profile: minimal
+  #         toolchain: stable
+  #         override: true
+  #     - name: Run WASM tests
+  #       run: wasm-pack test --node -- --workspace
```

Comment:
This is intentional because now tests are living in a separate library which needs to go under workspace so new changes lead to compile the solc bindings to wasm as well which is not possible right now as `wasm32-unknown-unknown` is not supporting c++ bindings right now. We did decided to comment the wasm-pack test cases on the CI and to make it work for now and for the releases we will manually run the wasm-pack test cases locally to verify whether everything is good or not.